apps/omg_watcher/lib/block_getter/core.ex

```diff
@@ -183,23 +187,30 @@ defmodule OMG.Watcher.BlockGetter.Core do
   """
   @spec get_blocks_to_apply(t(), list(), non_neg_integer()) ::
           {list({Block.t(), non_neg_integer()}), non_neg_integer(), list(), t()}
-  def get_blocks_to_apply(state, block_submitted_events, coordinator_height)
+  def get_blocks_to_apply(
```

Comment:
the `@doc` looks like it could use refreshing - digest the behavior more accurately